docs/launch_spec_experiments.py

```diff
@@ -0,0 +1,147 @@
+import os
+import sys
+from uuid import UUID
+
+from gem5art.artifact import Artifact
+from gem5art.run import gem5Run
+from gem5art.tasks.tasks import run_gem5_instance
+
+experiments_repo = Artifact.registerArtifact(
+    command = '',
+    typ = 'git repo',
+    name = 'experiment',
+    path =  './',
+    cwd = './',
+    documentation = 'local repo to run spec with gem5'
+)
+
+gem5_repo = Artifact.registerArtifact(
+    command = '''
+git clone https://gem5.googlesource.com/public/gem5;
```

Comment:
It's OK to have this tabbed over. I included code to strip the leading whitespace.